docs/toml.md

```diff
@@ -551,14 +551,17 @@ address = ":8080"
 # To enable basic auth on the webui
 # with 2 user/pass: test:test and test2:test2
 # Passwords can be encoded in MD5, SHA1 and BCrypt: you can use htpasswd to generate those ones
+# Users can be specified directly, or indirectly by referencing a file; if both are provided, the two are merged, with file contents having precedence
```

Comment:
Apologies, I got confused.

Maybe it makes sense to distinguish the two file types more clearly in the documentation by extending the sentence in line 554 like (**emphasis** by me) 

>Users can be specified directly **in the toml file**, or indirectly by referencing an **external** file;

WDYT?